iphone/Maps/Classes/MapsAppDelegate.mm

```diff
@@ -556,6 +554,9 @@ - (void)applicationWillResignActive:(UIApplication *)application
 {
   [self.mapViewController.appWallAd close];
   [RouteState save];
+  
+  [self.locationManager onBackground];
+  [self.mapViewController onEnterBackground];
```

Comment:
На чем сэкономит батарею? Если паузить рендеринг пока пользователь видит алерт диалог или принимает входящий звонок, или смотрит нотификацию, или еще много разных примеров, это позволит не гонять зря cpu и gpu.

> On Mar 21, 2016, at 13:23, Roman Kuznetsov <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/MapsAppDelegate.mm <https://github.com/mapsme/omim/pull/2365#discussion_r56802004>:
> 
> > @@ -556,6 +554,9 @@ - (void)applicationWillResignActive:(UIApplication *)application
> >  {
> >    [self.mapViewController.appWallAd close];
> >    [RouteState save];
> > +  
> > +  [self.locationManager onBackground];
> > +  [self.mapViewController onEnterBackground];
> Давайте для начала попробуем обработать правильный запуск дрейпа.
> Дрейп запускается верно. У меня возникает иногда ощущение, что ты забываешь, о чем мы уже 2 раза сегодня говорили, без обид :)
> я не вижу смысла синхронно ждать в случае SetRenderingEnabled(true)
> А я вижу здесь смысл. Этот вызов разблокирует заблокированное ранее.
> 
> Это еще и батарею сэкономит.
> Это на чем же? :)
> 
> Жди распила EnterBackground/Foreground. Все будет
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2365/files/4c1ad60019dfa841acbca6cd814eb2661ea98b14#r56802004>